packages/core/src/fhir-deduplication/__tests__/group-same-diag-reports.test.ts

```diff
@@ -49,11 +123,62 @@ describe("groupSameDiagnosticReports", () => {
     expect(diagReportsMap.size).toBe(2);
   });
 
+  it("discards diagReport if result and presented form are not present", () => {
+    diagReport.effectiveDateTime = dateTime.start;
+    diagReport.performer = [practRef];
+
+    const { diagReportsMap } = groupSameDiagnosticReports([diagReport]);
+    expect(diagReportsMap.size).toBe(0);
+  });
+
+  it("keeps diagReport if result is present and presentedForm is not - without a date", () => {
+    diagReport.performer = [practRef];
+
+    diagReport.result = resultExample;
+
+    const { diagReportsMap } = groupSameDiagnosticReports([diagReport]);
+    expect(diagReportsMap.size).toBe(1);
+
+    const dedupedDiagReport = diagReportsMap.values().next().value as DiagnosticReport;
+    expect(dedupedDiagReport.result).toBeTruthy();
+    expect(dedupedDiagReport.presentedForm).toBeFalsy();
+  });
+
+  it("keeps diagReport if result is present and presentedForm is not", () => {
+    diagReport.effectiveDateTime = dateTime.start;
+    diagReport.performer = [practRef];
+
+    diagReport.result = resultExample;
```

Comment:
Where's the `delete diagReport.presentedForm;`?

Reply:
i realized i had added this to the beforeEach:
```  
diagReport = makeDiagnosticReport({ id: diagReportId, result: resultExample });
diagReport2 = makeDiagnosticReport({ id: diagReportId2, presentedForm: presentedFormExample });
```
and this was causing confusion for me, even tho im the one who's developing these tests. So i took out the result and presentedForm from the beforeEach, and now if it `result` or `presentedForm` isn't set specifically in the test, then it's already not present, so no need to delete it

Reply:
put it on the `beforeEach`